recipe/meta.yaml

```diff
@@ -28,6 +28,10 @@ build:
 
 requirements:
   build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - gnuconfig  # [unix]
```

Comment:
You need to copy the `build` requirements to all outputs, then it should work.

Reply:
ok, thanks.

Reply:
@xhochy, can you explain the inclusion of `gnuconfig` here?

Reply:
I presume it's because the tarball has a `gnuscripts/` directory in there - it isn't used so I have removed that extra requirement.

Reply:
Yes, there is probably `configure` / `config.guess` / `config.sub` in the tarball.